Add 0.62 blog post (#1778)
facebook/react-native-website: website/blog/2020-03-26-version-0.62.md

---
title: Announcing React Native 0.62 with Flipper
author: Rick Hanlon
authorTitle: React Native Core at Facebook
authorURL: https://twitter.com/rickhanlonii
authorImageURL: https://avatars3.githubusercontent.com/u/2440089?s=460&v=4
authorTwitter: rickhanlonii
category: announcements
---

Today we’re releasing React Native version 0.62 which includes support for Flipper by default.

> This release comes in the midst of a global pandemic. We’re releasing this version today to respect the work of hundreds of contributors who made this release possible and to prevent the release from falling too far behind master. Please be mindful of the reduced capacity of contributors to help with issues and prepare to delay upgrading if necessary.

Reviewer: ❤️

## Flipper by default

[Flipper](https://fbflipper.com/) is a developer tool for debugging mobile apps. It’s popular in the Android and iOS communities, and in this release we’ve enabled support by default for new and existing React Native apps.

![Screenshot of Flipper for React Native](/blog/assets/0.62-flipper.png)

Flipper provides the following features out of the box:

> - **Metro Actions**: Reload the app and triggering the Dev Menu right from the toolbar.

Reviewer 2: ```suggestion
- **Metro Actions**: Reload the app and trigger the Dev Menu right from the toolbar.
```

> - **Crash Reporter**: View the crash reports from Android and iOS devices for all crashes.

Reviewer 2: ```suggestion
- **Crash Reporter**: View crash reports from Android and iOS devices.
```

Unsure if the “all crashes” had any significance?

Author: Honestly was just fluffing the content so the line ended close to all the others

Reviewer 2: Ahaha, I can respect that 😄 

![](https://media.giphy.com/media/a9wFQSc0oRQGI/giphy.gif)

> - **React DevTools**: Use the newest version React DevTools right along side all your other tools.

Reviewer 2: ```suggestion
- **React DevTools**: Use the newest version of React DevTools right alongside all your other tools.
```

> - **Network Inspector**: View all of the network requests made by the device or React Native app.

Reviewer 2: What does “requests made by the device” mean? Is it about network requests not triggered by the app? And is that really the case? (I assumed the Flipper plugin ties into RN’s network stack, not lower-level.)

Author: It hooks into the network stack of the app so it shows any requests, not just the ones from React Native (e.g. hybrid apps)

Reviewer 3: Should be application level indeed, not device level

Reviewer 3: Good point, maybe:  `made by the application`?

Reviewer 2: Yeah, I think that sounds good 👍

Reviewer 2: ```suggestion
- **Network Inspector**: View all of the network requests made by the application.
```

Author: ```suggestion
- **Network Inspector**: View all of the network requests made by the application.
```

> - **Metro and Device Logs**: View, search, and filter all of logs from both Metro and the Device.

Reviewer 2: ```suggestion
- **Metro and Device Logs**: View, search, and filter all logs from both Metro and the app.
```

?

Author: I believe this shows the logs from the device _and_ from the app that's running since it hooks into ADB/logcat.

Reviewer 3: Correct, it essence: it shows all logs from the connected devices, and all logs of metro

Reviewer 2: Gotcha, so all ‘console’ logs?

Reviewer 2: So then just this?

```suggestion
- **Metro and Device Logs**: View, search, and filter all logs from both Metro and the Device.
```

As part of our [Lean Core effort](https://reactnative.dev/blog/#lean-core) and to bring Apple TV in line with other platforms like React Native Windows and React Native macOS, we’ve started to remove Apple TV specific code from core.

Going forward, Apple TV support for React Native will be maintained in [react-native-community/react-native-tvos](https://github.com/react-native-community/react-native-tvos) along with the corresponding `react-native-tvos` NPM package. This is a full fork of the main repository, with only the changes needed to support Apple TV.

Releases of `react-native-tvos` will be based on the public release of React Native. For this 0.62 release of `react-native` please upgrade Apple TV projects to use `react-native-tvos` 0.62.

## Upgrade Support

When 0.61 was released, the community introduced a new [upgrade helper](https://react-native-community.github.io/upgrade-helper/) tool to support developers upgrading to new versions of React Native. The upgrade helper provides a diff of changes from the version you're on to the version you're targeting, allowing you to see the changes that need to be made for your specific upgrace.

> Even with this tool, issues come up when upgrading. Today we're introducting more dedicated upgrading support by announcing [Upgrade-Support](https://github.com/react-native-community/upgrade-support). Upgrade Support is a GitHub issue tracker where users can submit issues specific to upgrading their projects to recieve help from the community.

Reviewer 2: ```suggestion
Even with this tool, issues come up when upgrading. Today we're introducting more dedicated upgrading support by announcing [Upgrade-Support](https://github.com/react-native-community/upgrade-support). Upgrade Support is a GitHub issue tracker where users can submit issues specific to upgrading their projects to receive help from the community.
```
